src/Comments.jsx

```diff
@@ -37,6 +39,20 @@ export default class _Comments extends React.Component {
 
   componentDidMount() {
     this._ismounted = true;
+    fetch(`${CUSTOMIZR_URL}/v1/settings?resourceId=http://comment.trdlnk.cimpress.io/`, {
```

Comment:
The resource id is being mentioned again and again. Maybe a const?

Reply:
I have put this into a field now